lib/sidekiq/job_retry.rb

```diff
@@ -71,6 +71,7 @@ class Skip < Handled; end
     def initialize(capsule)
       @config = @capsule = capsule
       @max_retries = Sidekiq.default_configuration[:max_retries] || DEFAULT_MAX_RETRY_ATTEMPTS
+      @backtrace_cleaner = Sidekiq.default_configuration[:backtrace_cleaner]
```

Comment:
Nice, I hadn't gone this far into cleaning up the backtrace of jobs to store :+1: